routing/index_router.cpp

```diff
@@ -640,15 +641,14 @@ IRouter::ResultCode IndexRouter::AdjustRoute(Checkpoints const & checkpoints,
   return IRouter::NoError;
 }
 
-WorldGraph IndexRouter::MakeWorldGraph()
+std::unique_ptr<WorldGraph> IndexRouter::MakeWorldGraph()
```

Comment:
Looks like std:: prefix is not used in the file?

Reply:
removed